x/bank/simulation/operations.go

```diff
@@ -220,6 +249,85 @@ func SimulateMsgMultiSend(ak types.AccountKeeper, bk keeper.Keeper) simtypes.Ope
 	}
 }
 
+// SimulateMsgMultiSendToModuleAccount sends coins to Module Accounts
+func SimulateMsgMultiSendToModuleAccount(ak types.AccountKeeper, bk keeper.Keeper) simtypes.Operation {
+	return func(
+		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
+		accs []simtypes.Account, chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+
+		inputs := make([]types.Input, 2)
+		outputs := make([]types.Output, 2)
+		// collect signer privKeys
+		privs := make([]cryptotypes.PrivKey, len(inputs))
+
+		var totalSentCoins sdk.Coins
+		for i := range inputs {
+
+			// Sender
+			simAcc := accs[i]
+
+			// set signer privkey
+			privs[i] = simAcc.PrivKey
+
+			spendable := bk.SpendableCoins(ctx, simAcc.Address)
+			coins := simtypes.RandSubsetCoins(r, spendable)
+
+			// set next input and accumulate total sent coins
+			inputs[i] = types.NewInput(simAcc.Address, coins)
+			totalSentCoins = totalSentCoins.Add(coins...)
+		}
+
+		// Check send_enabled status of each sent coin denom
+		if err := bk.SendEnabledCoins(ctx, totalSentCoins...); err != nil {
+			return simtypes.NoOpMsg(types.ModuleName, types.TypeMsgMultiSend, err.Error()), nil, nil
+		}
+
+		for o := range outputs {
+
+			to := accs[o+len(inputs)]
```

Comment:
how do you know that `to` is a Module account? The function name  suggests that we should send to a Module account, but we don't have any assumption, nor any relation to it in the code. So either let's remove the function name or hardcode some module accounts here.

Reply:
[setupAccounts](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/8818-use-module-accs/x/bank/simulation/operations_test.go#L199) sets up 2 accounts and 2 module accounts at the end.